### PR TITLE
Added buttons for actions style

### LINKED ISSUE #221: Style all actions into buttons
On the following pages, add styling to the "Edit", "Destroy", "Delete" buttons.

### `/carrer/#`
image

### `/categories`
image

### `/categories/#`
image

### `/locations`
image

### `/agreements`
image

### `/agreements/1` and `/agreements/2`
image

### `/fee_types`
image

### `/fee_types/#`
image

### `/membership_types/new`
image

### PR DESCRIPTION
**Pull request by DouglasLutz**
Resolves #221 

### Description

* Altered the actions of the following pages into buttons:
/carrer/#
/categories
/categories/#
/locations
/agreements
/agreements/#
/fee_types
/fee_types/#
/membership_types
/membership_types/new

* Altered the structure of the table(merged the buttons rows into only one and aligned it to the right) in pages that have a list of components to match the same layout in all views(shown in screenshots)

### Type of change

* Bug fix (non-breaking change which fixes an issue)

### How Has This Been Tested?

Running rubocop and rspec after the modifications.

#### /carrier/# ####

Before:
before-carriers-#

After:
after-carrier-#

#### /categories ####

Before:
before-categories

After:
after-categories

### /categories/# ###

Before:
before-categories-#

After:
after-categories-#

### /locations ###

Before:
before-locations

After:
after-locations

### /agreements ###

Before:
before-agreements

After:
after-agreements

### /agreements/# ###

Before:
before-agreements-#

After:
after-agreements-#

### /fee_types ###

Before:
before-fee-types

After:
after-fee-types

### /fee_types/# ###

Before:
before-fee-types-#

After:
after-fee-types-#

### /membership_types ###

Before:
before-membership-types

After:
after-membership-types

### /membership_types/new ###

Before:
before-membership-types-new

After:
after-membership-types-new

### CODE REVIEW

File: app/views/agreements/index.html.erb [hunk 1]
```diff
@@ -25,8 +24,10 @@
             <td><%= link_to agreement.title, agreement %></td>
             <td><%= truncate(agreement.content) %></td>
 
-            <td><%= link_to 'Edit', edit_agreement_path(agreement) %></td>
-            <td><%= link_to 'Destroy', agreement, method: :delete, data: { confirm: 'Are you sure?' } %></td>
+            <td align="right">
+                <%= link_to 'Edit', edit_agreement_path(agreement), class: "btn btn-primary btn-sm" %> |
```
Comment (mauricio-prl) [inline app/views/agreements/index.html.erb]: I think you could use single quotes, example:
 ```ruby
  class: 'btn btn-primary btn-sm'
```
Once it hasn't strings interpolation.

Comment (DouglasLutz) [reply app/views/agreements/index.html.erb]: Yeah, sure

Comment (mauricio-prl) [reply app/views/agreements/index.html.erb]: Nice! Thanks!

**Review by dixpac — APPROVED** [review]
Thanks @DouglasLutz!

**Pull request merged**